velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -3324,6 +3324,27 @@ TEST_F(DateTimeFunctionsTest, timeZoneMinute) {
       "Unable to parse timestamp value: \"2023-\", expected format is (YYYY-MM-DD HH:MM:SS[.MS])");
 }
 
+TEST_F(DateTimeFunctionsTest, fromISO8601FunctionDate) {
+  const auto fromISODate =
+      [&](const std::optional<std::string>& isoDateString) {
+        return evaluateOnce<int32_t>("from_iso8601_date(c0)", isoDateString);
+      };
+
+  EXPECT_EQ(0, fromISODate("1970-01-01T03:19:58.000"));
+  EXPECT_EQ(18297, fromISODate("2020-02-05T14:27:39.000-07:00"));
```

Comment:
Maybe add tests for some more extreme values like 9999-12-31 and 0000-01-01

Reply:
Good point, added.